ci/tests/general/general_alba.py

```diff
@@ -630,36 +630,56 @@ def filter_disks(disk_names, amount, disk_type):
         :param disk_type: Type of disk
         :return: Filtered disks
         """
-        count = 0
         node_ids = []
-        filtered_disks = dict()
```

Comment:
I'd use a set for node_ids as these should be unique and the length should equal the nr of nodes